infra/base-images/base-runner/run_fuzzer

```diff
@@ -21,6 +21,18 @@
 export PATH=$OUT:$PATH
 cd $OUT
 
+# Allow to specify the number of jobs for fuzzers that support this natively (i.e., libfuzzer, honggfuzz)
+N_JOBS=${N_JOBS:-}
```

Comment:
I don't think we should expose both of these since one of them is a libfuzzer implementation detail (that jobs and workers aren't the same). I would just have one var N_JOBS and use it for both. To be honest, I think it's pretty rare for someone to want these to be different for libFuzzer.

Reply:
Good point.